eslint-plugin-prettier.js

```diff
@@ -224,4 +216,30 @@ const eslintPluginPrettier = {
   },
 };
 
+// Assign configs after initial plugin configuration, as flat configs
+// require referencing the original plugin that contains the rules.
+Object.assign(eslintPluginPrettier.configs, {
```

Comment:
How about `eslintPluginPrettier.configs = ` instead?

Reply:
Yeah that should be fine now since the original `eslintPluginPrettier.configs` object is empty. Updated it now!